mimic/test/dummy.py

```diff
@@ -96,63 +101,61 @@ def resource(self):
         return example_resource
 
 
-@implementer(IEndpointTemplate)
-class ExampleEndpointTemplate(object):
+def ExampleEndpointTemplate(name=u"example", region="EXTERNAL", version="v1",
+                            url="https://api.external.example.com:8080",
+                            public_url=None, internal_url=None, admin_url=None,
+                            version_info_url=None, version_list_url=None,
+                            type_id=u"example", enabled=False,
+                            uuid=text_type(uuid.uuid4()),
```

Comment:
This is fixed at import time.  If that's the idea, then having it fixed across all runs would be good; define it as something like `uuid.UUID('4397012d-4aab-4437-9ac5-b123744a3e80')`.  If the idea is that the unspecified default should generate a new, unique UUID each time, be sure to do that in the function.